README.md

```diff
@@ -4,7 +4,7 @@ A GitHub action that runs a phpcs and reports the result to reviewdog
 # Prerequisites
 * Composer install has been executed
 * Phpcs is install by composer and available in the composer config bin-dir
-* The code styles used by phpcs are installed and confgured in .phpcs.xml
+* The code styles used by phpcs are installed and confgured in phpcs.xml
```

Comment:
PHPCS support both `phpcs.xml` and `.phpcs.xml`. I personally prefer having these kind of files hidden per default (phpunit, I'm looking at you), so I'd suggest keeping the dot to encourage that.

Reply:
Check